standard/coordinates.md

```diff
@@ -0,0 +1,412 @@
+---
+# LaTeX frontmatter equivalent to newcommand https://mystmd.org/guide/math#math-macros
+math:
+  '\bfV': '{\bf V}'
+---
+
+(c:coords)=
+# Coordinates
+
+%-----------------------------------------------------------------------------
+(s:coords)=
+## Coordinate Systems
+
+```{figure} figures/coordinates.svg
+:width: 70%
+:label: f:coords
+:name: Floor coordinate system
+
+The `floor` coordinate system
+is independent of the accelerator.  The `branch` curvilinear coordinate system follows the bends
+of the accelerator. The `branch reference curve` is the {math}`x = y = 0` curve of the curvilinear coordinate
+system. Each lattice element has `element body` coordinates which, if the element has no
+alignment shifts (not "misaligned"), is the same as the `branch` coordinates.
+```
+
+%-----------------------------------------------------------------------------
+
+The lattice standard uses three coordinate systems as illustrated in the figure above. 
+First, the `floor` coordinates are coordinates independent of the accelerator.
+The position of the accelerator itself as well as external objects like the building the
+accelerator is in  may be described using `floor` coordinates.
+
+It is inconvenient to describe the position lattice elements and the position of a 
+particle beam using the `floor` coordinate system so, for each tracking lattice branch,
+a [](#s:branch.coords}  is used. This curvilinear coordinate
+system defines the nominal position of the lattice elements. The relationship between the
+`branch` and `floor` coordinate systems is described in section [](#s:floor). 
+Lord (non-tracking) branches do not have an associated branch coordinate system.
+
+The `branch reference curve` is the {math}`x = y = 0` curve of the curvilinear coordinate system. 
```

Comment:
Maybe it would be good here to explain that `x` is a horizontal coordinate, while `y` is vertical.

Reply:
Will be in new revision.